src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
Ideally, it would be better to put such variables in an env file
You create the same constant with the same url in all your api files, it should be the same common for all